src/LeanCode.ContractsGenerator/Analyzers/BaseAnalyzer.cs

```diff
@@ -20,17 +20,17 @@ public virtual IEnumerable<AnalyzeError> AnalyzeValueRef(AnalyzerContext context
         return Enumerable.Empty<AnalyzeError>();
     }
 
-    public virtual IEnumerable<AnalyzeError> AnalyzeGenericTypeRef(AnalyzerContext context, TypeRef typeRef, TypeRef.Types.Generic g)
+    public virtual IEnumerable<AnalyzeError> AnalyzeGenericTypeRef(AnalyzerContext context, TypeRef.Types.Generic g)
```

Comment:
Why the change? This makes `AnalyzeGenericTypeRef` have _less_ data than it needs to have - it does not have thea `TypeRef.nullable`.